dev/pre-commit

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+make format-xml
+git add en de
```

Comment:
Isn't that problematic though? IIUC this would level-out any differences incurred from the auto-formatter by updating the index already. So now I cannot discern my (presumably manual) changes from the hooked changes anymore.

If you would just omit the `git add`, then I could still distinguish the index from the new state of the working directory (`git diff` them).

In fact, that difference is exactly what the line below should check on – instead of distrusting _any_ changes to the `*.dita` files (which would prevent one from ever changing them manually again, short of circumventing the hook with `--no-verify`).